### PR TITLE
merged packets, 3rd party branches, and added support for phg parsing

### PR DESCRIPTION
**Pull request by stef**
i cherry picked and tested some of the branches.i run this in production.

### CODE REVIEW

**Comment by hemna** [comment]
aprslib should remove py 2.7 tests as python 2 is dead.

File: aprslib/parsing/thirdparty.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        'parse_thirdparty',
+        ]
+
+def parse_thirdparty(body):
+    parsed = {'format':'Third party'}
+
+    # Parse sub-packet
+    try:
+        subpacket = parse(body)
+    except (UnknownFormat) as ukf:
```
Comment (hemna) [inline aprslib/parsing/thirdparty.py:16]: Should add this patch as well,
https://github.com/wsmitchell3/aprs-python/commit/441edbfeedf477bd307a622b572b525b181266b2

**Comment by TheCranston** [comment]
From #62 I've updated the feedback.  The PHG is supposed to be 4 decimals and the 1.2 spec allows for extension into PHGR of 5 decimals but is terminated by a "/" as to maintain backwards compatibility.  So no "/" means only grab four decimals from the string and the rest is freeform data including comments, etc.  Presence of a "/" at position 6 after PHG indicates PHGR and should be parsed differently.

**Comment by rossengeorgiev** [comment]
This PR is too busy. I'm going to close it favor of individual PR for fixes and enhancements.

**Pull request closed without merging**